.gitignore

```diff
@@ -19,3 +19,6 @@ src/renderer/licenses.ts
 
 # Missed Translations during Development
 *.missing.json
+
+.idea/*
+.gitignore
```

Comment:
We don't add `.gitignore` to the ignore file because we actually want to track the ignore file.